ru.linux_gaming.PortProton.yml

```diff
@@ -619,6 +619,7 @@ modules:
         install -Dm644 ru.linux_gaming.PortProton.svg /app/share/icons/hicolor/scalable/apps/ru.linux_gaming.PortProton.svg
         install -Dm644 ru.linux_gaming.PortProton.desktop /app/share/applications/ru.linux_gaming.PortProton.desktop
         install -Dm644 ru.linux_gaming.PortProton.metainfo.xml /app/share/metainfo/ru.linux_gaming.PortProton.metainfo.xml
+        install -Dm644 LICENSE /app/share/licenses/ru.linux_gaming.PortProton/LICENSE
```

Comment:
This does not include the licenses of the components you are using. 

For example, [this line](https://github.com/Castro-Fidel/wine_builds/blob/517472b9f86b67ee95c2cd5ea69d57d80928c560/04_ge_to_lg.sh#L26C9-L26C127) is downloading https://github.com/GloriousEggroll/proton-ge-custom/releases/download/GE-Proton9-5/GE-Proton9-5.tar.gz which has its own LICENSE file in it but that is nowehere in your tarball https://github.com/Castro-Fidel/wine_builds/releases/download/PROTON_LG_9-4/PROTON_LG_9-4.tar.xz